00:00 +0: loading /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart                                                                                          Running Gradle task 'assembleDebug'...                          00:01 +0: loading /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart                                                                                          00:02 +0: loading /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart                                                                                          00:03 +0: loading /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart                                                                                          00:04 +0: loading /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart                                                                                          00:05 +0: loading /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart                                                                                          00:06 +0: loading /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart                                                                                          00:07 +0: loading /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart                                                                                              7.2s
✓ Built build/app/outputs/flutter-apk/app-debug.apk
Installing build/app/outputs/flutter-apk/app-debug.apk...       00:08 +0: loading /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart                                                                                          00:09 +0: loading /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart                                                                                           1,987ms
00:10 +0: loading /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart                                                                                          00:11 +0: loading /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart                                                                                          00:11 +0: 项目管理流程测试 创建项目并添加任务                                                                                                                                                                           00:12 +0: 项目管理流程测试 创建项目并添加任务                                                                                                                                                                           00:13 +0: 项目管理流程测试 创建项目并添加任务                                                                                                                                                                           00:13 +0: 项目管理流程测试 创建项目并添加任务                                                                                                                                                                           
══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
The following TestFailure was thrown running a test:
Expected: exactly one matching candidate
  Actual: _TextContainingWidgetFinder:<Found 3 widgets with text containing 项目: [
            Text("工作相关项目和任务", overflow: ellipsis, maxLines: 1, dependencies: [DefaultSelectionStyle,
DefaultTextStyle, MediaQuery]),
            Text("活动项目", dependencies: [DefaultSelectionStyle, DefaultTextStyle, MediaQuery]),
            Text("项目", debugLabel: ((englishLike labelMedium 2021).merge((whiteMountainView
labelMedium).apply)).apply, inherit: false, color: Color(alpha: 1.0000, red: 0.8824, green: 0.9137,
blue: 0.9569, colorSpace: ColorSpace.sRGB), family: Roboto, size: 12.0, weight: 500, letterSpacing:
0.5, baseline: alphabetic, height: 1.3x, leadingDistribution: even, decoration: Color(alpha: 1.0000,
red: 0.8824, green: 0.9137, blue: 0.9569, colorSpace: ColorSpace.sRGB) TextDecoration.none,
dependencies: [DefaultSelectionStyle, DefaultTextStyle, MediaQuery]),
          ]>
   Which: is too many

When the exception was thrown, this was the stack:
#4      main.<anonymous closure>.<anonymous closure> (file:///Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart:20:7)
<asynchronous suspension>
#5      testWidgets.<anonymous closure>.<anonymous closure> (package:flutter_test/src/widget_tester.dart:193:15)
<asynchronous suspension>
#6      TestWidgetsFlutterBinding._runTestBody (package:flutter_test/src/binding.dart:1064:5)
<asynchronous suspension>
<asynchronous suspension>
(elided one frame from package:stack_trace)

This was caught by the test expectation on the following line:
  file:///Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart line 20
The test description was:
  创建项目并添加任务
════════════════════════════════════════════════════════════════════════════════════════════════════
00:13 +0 -1: 项目管理流程测试 创建项目并添加任务 [E]                                                                                                                                                                    
  Test failed. See exception logs above.
  The test description was: 创建项目并添加任务
  

To run this test again: /Users/weikuang/development/flutter/bin/cache/dart-sdk/bin/dart test /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart -p vm --plain-name '项目管理流程测试 创建项目并添加任务'
00:13 +0 -1: 项目管理流程测试 编辑和删除项目                                                                                                                                                                          00:14 +0 -1: 项目管理流程测试 编辑和删除项目                                                                                                                                                                          00:14 +0 -1: 项目管理流程测试 编辑和删除项目                                                                                                                                                                          
══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
The following assertion was thrown running a test:
The finder "Found 2 widgets with icon "IconData(U+0E047)": [
  Icon(IconData(U+0E047), dependencies: [Directionality, IconTheme]),
  Icon(IconData(U+0E047), dependencies: [Directionality, IconTheme]),
]" (used in a call to "tap()") ambiguously found multiple matching widgets. The "tap()" method needs
a single target.

When the exception was thrown, this was the stack:
#0      WidgetController._getElementPoint (package:flutter_test/src/controller.dart:2014:7)
#1      WidgetController.getCenter (package:flutter_test/src/controller.dart:1861:12)
#2      WidgetController.tap (package:flutter_test/src/controller.dart:1041:7)
#3      main.<anonymous closure>.<anonymous closure> (file:///Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart:75:20)
<asynchronous suspension>
#4      testWidgets.<anonymous closure>.<anonymous closure> (package:flutter_test/src/widget_tester.dart:193:15)
<asynchronous suspension>
#5      TestWidgetsFlutterBinding._runTestBody (package:flutter_test/src/binding.dart:1064:5)
<asynchronous suspension>
<asynchronous suspension>
(elided one frame from package:stack_trace)

The test description was:
  编辑和删除项目
════════════════════════════════════════════════════════════════════════════════════════════════════
00:14 +0 -2: 项目管理流程测试 编辑和删除项目 [E]                                                                                                                                                                      
  Test failed. See exception logs above.
  The test description was: 编辑和删除项目
  

To run this test again: /Users/weikuang/development/flutter/bin/cache/dart-sdk/bin/dart test /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart -p vm --plain-name '项目管理流程测试 编辑和删除项目'
00:14 +0 -2: 项目管理流程测试 项目任务统计和进度                                                                                                                                                                        00:15 +0 -2: 项目管理流程测试 项目任务统计和进度                                                                                                                                                                        00:16 +0 -2: 项目管理流程测试 项目任务统计和进度                                                                                                                                                                        00:16 +0 -2: 项目管理流程测试 项目任务统计和进度                                                                                                                                                                        
══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
The following assertion was thrown running a test:
The finder "Found 2 widgets with icon "IconData(U+0E047)": [
  Icon(IconData(U+0E047), dependencies: [Directionality, IconTheme]),
  Icon(IconData(U+0E047), dependencies: [Directionality, IconTheme]),
]" (used in a call to "tap()") ambiguously found multiple matching widgets. The "tap()" method needs
a single target.

When the exception was thrown, this was the stack:
#0      WidgetController._getElementPoint (package:flutter_test/src/controller.dart:2014:7)
#1      WidgetController.getCenter (package:flutter_test/src/controller.dart:1861:12)
#2      WidgetController.tap (package:flutter_test/src/controller.dart:1041:7)
#3      main.<anonymous closure>.<anonymous closure> (file:///Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart:131:20)
<asynchronous suspension>
#4      testWidgets.<anonymous closure>.<anonymous closure> (package:flutter_test/src/widget_tester.dart:193:15)
<asynchronous suspension>
#5      TestWidgetsFlutterBinding._runTestBody (package:flutter_test/src/binding.dart:1064:5)
<asynchronous suspension>
<asynchronous suspension>
(elided one frame from package:stack_trace)

The test description was:
  项目任务统计和进度
════════════════════════════════════════════════════════════════════════════════════════════════════
00:16 +0 -3: 项目管理流程测试 项目任务统计和进度 [E]                                                                                                                                                                    
  Test failed. See exception logs above.
  The test description was: 项目任务统计和进度
  

To run this test again: /Users/weikuang/development/flutter/bin/cache/dart-sdk/bin/dart test /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart -p vm --plain-name '项目管理流程测试 项目任务统计和进度'
00:16 +0 -3: 项目管理流程测试 在不同视图中查看项目任务                                                                                                                                                                     00:17 +0 -3: 项目管理流程测试 在不同视图中查看项目任务                                                                                                                                                                     00:17 +0 -3: 项目管理流程测试 在不同视图中查看项目任务                                                                                                                                                                     
══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
The following assertion was thrown running a test:
The finder "Found 2 widgets with icon "IconData(U+0E047)": [
  Icon(IconData(U+0E047), dependencies: [Directionality, IconTheme]),
  Icon(IconData(U+0E047), dependencies: [Directionality, IconTheme]),
]" (used in a call to "tap()") ambiguously found multiple matching widgets. The "tap()" method needs
a single target.

When the exception was thrown, this was the stack:
#0      WidgetController._getElementPoint (package:flutter_test/src/controller.dart:2014:7)
#1      WidgetController.getCenter (package:flutter_test/src/controller.dart:1861:12)
#2      WidgetController.tap (package:flutter_test/src/controller.dart:1041:7)
#3      main.<anonymous closure>.<anonymous closure> (file:///Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart:197:20)
<asynchronous suspension>
#4      testWidgets.<anonymous closure>.<anonymous closure> (package:flutter_test/src/widget_tester.dart:193:15)
<asynchronous suspension>
#5      TestWidgetsFlutterBinding._runTestBody (package:flutter_test/src/binding.dart:1064:5)
<asynchronous suspension>
<asynchronous suspension>
(elided one frame from package:stack_trace)

The test description was:
  在不同视图中查看项目任务
════════════════════════════════════════════════════════════════════════════════════════════════════
00:17 +0 -4: 项目管理流程测试 在不同视图中查看项目任务 [E]                                                                                                                                                                 
  Test failed. See exception logs above.
  The test description was: 在不同视图中查看项目任务
  

To run this test again: /Users/weikuang/development/flutter/bin/cache/dart-sdk/bin/dart test /Users/weikuang/Apps/gtd4_without_clean_achitecture/integration_test/project_flow_test.dart -p vm --plain-name '项目管理流程测试 在不同视图中查看项目任务'
00:17 +0 -4: (tearDownAll)                                                                                                                                                                             00:17 +0 -4: Some tests failed.                                                                                                                                                                        
